ompi/mca/coll/han/coll_han_subcomms.c

```diff
@@ -0,0 +1,463 @@
+/*
+ * Copyright (c) 2004-2007 The Trustees of Indiana University and Indiana
```

Comment:
Why the entire set of copyrights ? Bull and UTK should be enough.

Reply:
The code is an evolution of this file (+ renamed and splitted) which contained these copyrights : https://github.com/bosilca/ompi/blob/future_adapt/ompi/mca/coll/future/coll_future_module.c

Reply:
our mistake to have added all the copyrights there. Please just keep Bull and UTK.

Reply:
Fixed in a746a13541eb11a76663d06be244011c8de2672c